src/neuroglancer/viewer.ts

```diff
@@ -83,30 +104,46 @@ export class Viewer extends RefCounted implements ViewerState {
   layerPanel: LayerPanel;
   layerSelectedValues =
       this.registerDisposer(new LayerSelectedValues(this.layerManager, this.mouseState));
-  worker = new RPC(new Worker('chunk_worker.bundle.js'));
   resetInitiated = new NullarySignal();
 
-  chunkQueueManager = new ChunkQueueManager(this.worker, this.display.gl, {
-    gpuMemory: new AvailableCapacity(1e6, 1e9),
-    systemMemory: new AvailableCapacity(1e7, 2e9),
-    download: new AvailableCapacity(32, Number.POSITIVE_INFINITY)
-  });
-  chunkManager = new ChunkManager(this.chunkQueueManager);
+  get chunkManager() {
+    return this.dataContext.chunkManager;
+  }
+  get chunkQueueManager() {
+    return this.dataContext.chunkQueueManager;
+  }
+
   keyMap = new KeySequenceMap();
   keyCommands = new Map<string, (this: Viewer) => void>();
-  layerSpecification = new LayerListSpecification(
-      this.layerManager, this.chunkManager, this.worker, this.layerSelectedValues,
-      this.navigationState.voxelSize);
+  layerSpecification: LayerListSpecification;
   layoutName = new TrackableValue<string>(LAYOUTS[0][0], validateLayoutName);
 
   state = new CompoundTrackable();
 
   private options: ViewerOptions;
 
-  constructor(public display: DisplayContext, options: ViewerOptions = {}) {
```

Comment:
It appears you need to add the showLayerDialog=false flag to options to suppress the initial overlay. It probably makes sense to change the check at 186 to include a check for dataContext and/or WatchableVisibilityPriority as well.